service/controller/clusterapi/v29/templates/cloudformation/tccp/main.go

```diff
@@ -6,10 +6,6 @@ AWSTemplateFormatVersion: 2010-09-09
 Description: Tenant Cluster Control Plane Cloud Formation Stack.
 Outputs:
   {{ template "outputs" . }}
-Parameters:
-  VersionBundleVersionParameter:
```

Comment:
Not sure if this should be kept as `OperatorVersion` or if this is not used anymore? I'd say spinning a cluster tells that.

Reply:
IMO this is some legacy bullshit we never really needed. I wanted to get rid of it and it looks like it was as useless as I though.